### ISSUE TITLE
Batch input to duckdb?

### ISSUE THREAD

**Issue by jeberly** [benthosdev/benthos#1789]
Is there an existing method to do batch inserts into duckdb? Or any plans for something like that? 

I would like to get logs into duckdb so we can analyze log data in a simple to host/maintain solution.

**Comment by jeberly** [comment]
Thanks! Good idea with copying an existing plugin.

**Comment by artemklevtsov** [comment]
Also you can use `command` process with the duckdb cli.

**Comment by rasnes** [comment]
Waiting for someone to implement a DuckDB driver with [purego](https://github.com/ebitengine/purego) so we can add it to Redpanda Connect ⌛ 😊

**Comment by Flamefork** [comment]
> Waiting for someone to implement a DuckDB driver with [purego](https://github.com/ebitengine/purego) so we can add it to Redpanda Connect ⌛ 😊

https://github.com/fpt/go-pduckdb

**Comment by twmb** [comment]
Closing — DuckDB support is blocked on a suitable Go driver without CGo. The go-pduckdb project was linked in the comments as a potential option. If a viable driver matures, feel free to open a new issue or PR.
